### PR TITLE
vjsstandard - Tests - Utils folder

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
lint all of the tests in the test/unit/utils folder

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Comment by misteroneill** [comment]
LGTM

File: test/unit/utils/fn.test.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-q.module('fn');
+QUnit.module('fn');
 
-test('should add context to a function', function(){
-  var newContext = { test: 'obj'};
-  var asdf = function(){
-    ok(this === newContext);
+QUnit.test('should add context to a function', function() {
+  const newContext = { test: 'obj'};
+  const asdf = function() {
```
Comment (gkatsev) [inline test/unit/utils/fn.test.js]: these are great function names...

File: test/unit/utils/format-time.test.js [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
-test('should format invalid times as dashes', function(){
-  equal(formatTime(Infinity, 90), '-:-');
-  equal(formatTime(NaN), '-:-');
-  // equal(formatTime(NaN, 216000), '-:--:--');
-  equal(formatTime(10, Infinity), '0:00:10');
-  equal(formatTime(90, NaN), '1:30');
+QUnit.test('should format invalid times as dashes', function() {
+  QUnit.equal(formatTime(Infinity, 90), '-:-');
+  QUnit.equal(formatTime(NaN), '-:-');
+  // QUnit.equal(formatTime(NaN, 216000), '-:--:--');
```
Comment (gkatsev) [inline test/unit/utils/format-time.test.js]: just delete this line.

**Comment by gkatsev** [comment]
LGTM

File: test/unit/utils/fn.test.js (hunk 1 above)
Comment (BrandonOCasey) [inline test/unit/utils/fn.test.js]: haha

**Pull request merged**